app/views/generic_files/_descriptions.html.erb

```diff
@@ -3,7 +3,7 @@
         <%= hidden_field_tag('redirect_tab', 'descriptions') %>
         <h1>Descriptions <span class="pull-right required"><abbr title="required">*</abbr> indicates required fields</span></h1>
         <div class="well">
-            <% form_terms = f.object.terms - [:degree_grantor, :date_submitted, :date_accepted] %>
+            <% form_terms = f.object.terms - [:degree_grantor, :doi, :date_submitted, :date_accepted] %>
```

Comment:
Shouldn't have variable assignment logic in the view, should be done in the GenericFileEditForm class.

This is required so DOI isn't an editable field in the descriptions.